src/extension.ts

```diff
@@ -122,14 +124,17 @@ export function activate(context: vscode.ExtensionContext) {
 	registerEvent(
 		'cosmosDB.onDidChangeConfiguration',
 		vscode.workspace.onDidChangeConfiguration,
-		async function
-			(this: IActionContext, event: vscode.ConfigurationChangeEvent): Promise<void> {
+		async function (this: IActionContext, event: vscode.ConfigurationChangeEvent): Promise<void> {
 			this.properties.isActivationEvent = "true";
 			this.suppressErrorDisplay = true;
 			if (event.affectsConfiguration(ext.settingsKeys.documentLabelFields)) {
 				await vscode.commands.executeCommand("cosmosDB.refresh");
 			}
 		});
+	registerCommand('cosmosDB.api.getDatabase', async () => {
+		const AttachedDatabaseTreeItemContextvalue: string = 'cosmosDBAttachDatabaseAccount';
```

Comment:
I don't think 'cosmosDBAttachDatabaseAccount' should be necessary?

Reply:
You're right, thanks!